src/components/Categories.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <>Categories</>;
```

Comment:
- [ ] In the Categories component, there should be a button. When clicked, it should display "Check status". Please add this button.